pkg/ebpf/telemetry/errors_collector_linux.go

```diff
@@ -28,7 +28,7 @@ const (
 // EBPFErrorsCollector implements the prometheus Collector interface
 // for collecting statistics about errors of ebpf helpers and ebpf maps operations.
 type EBPFErrorsCollector struct {
-	*EBPFTelemetry
+	T                *EBPFTelemetry
```

Comment:
Can we come up with a better name here? Maybe just `Telemetry`?

Reply:
I actually plan to make this unexported in the following PR, kept it exported for now to avoid making all the changes in a single PR